.travis.yml

```diff
@@ -9,12 +9,17 @@ python:
   - "2.7"
   - "3.5"
 
+env:
+  - TEST_DIR=root TEST_TYPE=lint
+  - TEST_DIR=root TEST_TYPE=unit
+  - TEST_DIR=utils
+
 install:
-  - pip install -r requirements.txt
```

Comment:
> For travis, we were installing packages outside of the tox venvs, and then also installing them again in each venv.

Yep, we are in agreement that ever since we migrated to `tox` this line is not useful here, because it applies to a different env.

Ideally we would be able to do as I started in the other comment, we could have `Ansible>=MIN_VERSION` in `requirements.txt`, and then have the tox config require more specific versions.

That may not work in the most straight-forward way because of https://github.com/pypa/pip/issues/988

```
$ cat requirements.txt 
Ansible>=2.1
$ pip install -r requirements.txt Ansible==2.2
Double requirement given: Ansible>=2.1 (from -r requirements.txt (line 1)) (already in Ansible==2.2, name='Ansible')
```

But if we call `pip` twice, it should work.